fetch/api/request/destination/fetch-destination-style.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../../../service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script>
+async_test(function(t) {
+    var worker_url = 'resources/fetch-destination-worker.js';
+    var scope = 'resources/empty.html';
+    var registration;
+
+    service_worker_unregister_and_register(t, worker_url, scope)
+      .then(t.step_func(function(r) {
+          registration = r;
+          return wait_for_state(t, r.installing, 'activated');
+        }))
+      .then(t.step_func(function() {
+          return with_iframe(scope);
+        }))
+      .then(t.step_func(function(frame) {
+          var node = frame.contentWindow.document.createElement("link");
+          node.rel = "stylesheet";
+          node.addEventListener("load", t.step_func_done(function() { registration.unregister(); }));
+          node.addEventListener("error", t.step_func_done(function() {
+              registration.unregister();
+              assert_unreached("Fetch destination style errored");
+          }));
+          node.href = "dummy?dest=style";
+          frame.contentWindow.document.body.appendChild(node);
+        }))
+      .catch(unreached_rejection(t));
+}, 'Fetch destination style');
```

Comment:
We should have a test for CSS `@import` which can produce a "style" destination as well.